tutorials/integrations/0xgasless.md

```diff
@@ -0,0 +1,141 @@
+
+1. Enter a name for your paymaster
+2. Select **Moonbeam** as the chain
+3. Press **Create**
+
+![Create Paymaster](/images/tutorials/integrations/0xgasless/0xgasless-1.webp)
+
+Your paymaster needs funds to cover gas fees for sponsored transactions. To deposit GLMR into your paymaster, take the following steps:
+
+1. Enter the amount you would like to deposit
+2. Press **Deposit** and confirm the transaction in your wallet
+
+![Fund Paymaster](/images/tutorials/integrations/0xgasless/0xgasless-2.webp)
+
+Your deposited funds remain flexible - use them to sponsor gasless transactions or withdraw them whenever needed.
+
+## Dispatching a Gasless Transaction
+
+In the following section, we'll create a script demonstrating how to dispatch a gasless transaction. 
+
+### Prerequisites
+
+Create a `.env` file in your project's root directory with the following:
+
+```bash
+PRIVATE_KEY=INSERT_PRIVATE_KEY
+RPC_URL=https://rpc.api.moonbeam.network
+```
+
+Why are we specifying a private key in the `.env`? While this transaction will be gasless, you still need a private key to sign the transaction. The account associated with this private key:
+
+- Does not need any GLMR tokens
+- Will not pay for gas fees
+- Is only used for transaction signing
+
+!!! note 
+
+	Never commit your .env file or share your private key. Add .env to your .gitignore file.
+
+Also, make sure you have installed the 0xGasless SDK and supporting `ethers` and `dotenv` packages:
+
+```bash
+npm install ethers dotenv @0xgasless/smart-account
+```
+
+First, we'll import the required packages as follows:
+
+```js
+require('dotenv').config();
+const ethers = require('ethers');
+const {
+  PaymasterMode,
+  createSmartAccountClient,
+} = require('@0xgasless/smart-account');
+```
+
+Next, we'll set the critical constants. We must define the `CHAIN_ID`, `BUNDLER_URL`, and `PAYMASTER_URL`. You can get your unique paymaster URL from the paymaster on your [0xGasless Dashboard](https://dashboard.0xgasless.com/paymaster){target=\_blank}.
+
+The contract address we've defined here is the address of an [Incrementer contract](https://moonscan.io/address/0x3ae26f2c909eb4f1edf97bf60b36529744b09213) on Moonbeam, on which we'll call the increment function specified by the function selector. This simple contract will allow us to easily see if the gasless transaction has been dispatched successfully. 
+
+```js
+const CHAIN_ID = 1284; // Moonbeam mainnet
+const BUNDLER_URL = `https://bundler.0xgasless.com/${CHAIN_ID}`;
+const PAYMASTER_URL =
+  'https://paymaster.0xgasless.com/v1/1284/rpc/INSERT_API_KEY';
+const CONTRACT_ADDRESS = '0x3aE26f2c909EB4F1EdF97bf60B36529744b09213';
+const FUNCTION_SELECTOR = '0xd09de08a';
+```
+
+#### Paymaster URL Formatting Note
+
+The Paymaster URL format has recently changed. Use:
+
+```
+https://paymaster.0xgasless.com/v1/1284/rpc/INSERT_API_KEY
+```
+
+Do not use the deprecated format:
```

Comment:
I'm not extremely fond of this? Maybe a warning note with the `!!!`?

Reply:
fixed!